packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -362,6 +363,7 @@ export class MgtFileList extends MgtTemplatedComponent {
   private _insightType: OfficeGraphInsightString;
   private _fileExtensions: string[];
   private _userId: string;
+  private _files: DriveItem[];
```

Comment:
`_files` seems to similar to `files`. Consider renaming it with a more descriptive name (ex `_preloadedFiles`)